central/processbaseline/service/service_impl_test.go

```diff
@@ -83,6 +105,7 @@ func TestProcessBaselineService(t *testing.T) {
 
 type ProcessBaselineServiceTestSuite struct {
 	suite.Suite
+	store     postgresStore.Store
```

Comment:
This is added to the `ProcessBaselineServiceTestSuite` struct so that it can be used in `fillDBDirect`.